CMakeLists.txt

```diff
@@ -625,10 +625,21 @@ if(WAVPACK_INSTALL_CMAKE_MODULE)
 		write_basic_package_version_file(wavpack-config-version.cmake COMPATIBILITY SameMajorVersion)
 	else()
 		write_basic_package_version_file(wavpack-config-version.cmake COMPATIBILITY SameMinorVersion)
-	endif()
+    endif()
```

Comment:
De-indentation here uncalled for. As far as I can see, the `configure_package_config_file` and `install` additions below should be indented.

Reply:
I've found an issue with the CMakeLists file. It mixes space and tab (and actually this line is properly indented if 1 tab is considered as 4 spaces). Maybe this issue should also be fixed later.

Reply:
My editor seemed to be confused about using what to indent when editing this line :-)